src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -6,7 +6,8 @@
         <h2>Company Name</h2>
       </header>
 
-      <div>** Correct Name component goes here **</div>
+      <NameRequestInfo />
+      <NameTranslations />
```

Comment:
Just porting over Sev's comments on other PR:

Our style is to use the PascalCase version of the component names here, so that VS Code highlights them nicely (and sometimes you can click on them to go to their definition).

Also, don't add divs around them if not needed.